pkgs/applications/editors/emacs/elisp-packages/manual-packages/ligo-mode/default.nix

```diff
@@ -34,10 +34,10 @@ melpaBuild {
 
   passthru.updateScript = unstableGitUpdater { };
 
-  meta = with lib; {
-    description = "A blazingly fast LSP client for Emacs";
-    homepage = "https://github.com/manateelazycat/lsp-bridge";
-    license = licenses.gpl3Only;
-    maintainers = with maintainers; [ ];
+  meta = {
+    description = "Emacs mode for LIGO smart-contract language";
```

Comment:
The upstream uses "A major mode for editing LIGO source code" https://gitlab.com/ligolang/ligo/-/blob/501abb19d2a6d198801c93b5879f43cb4fcef569/tools/emacs/ligo-mode.el#L2